owlkettle/widgets.nim

```diff
@@ -1320,7 +1346,9 @@ renderable MenuButton of BaseWidget:
   child: Widget
   popover: Widget
   
-  style: set[ButtonStyle]
+  style: set[ButtonStyle] ## Applies special styling to the button. 
+  ## May be one of `ButtonSuggested`, `ButtonDestructive`, `ButtonFlat`, `ButtonPill` or `ButtonCircular`.
+  ## Consult the ## [GTK4 documentation](https://developer.gnome.org/hig/patterns/controls/buttons.html?highlight=button#button-styles) for guidance on what to use.
```

Comment:
Multiline comments are currently not transferred to `widgets.md`. Just convert this to a single line.